package.json

```diff
@@ -9,7 +9,7 @@
     "compress": "^0.99.0",
     "debug": "^4.1.1",
     "httpntlm-maa": "^2.0.6",
-    "lodash": "^4.17.20",
+    "lodash": "^4.17.21",
```

Comment:
I think the change in this line is not needed. With the `^`, it will install the latest 4.x version.